module-apps/apps-common/widgets/ProgressTimerWithBarGraphAndCounter.cpp

```diff
@@ -27,7 +27,7 @@ namespace app
         if (text == nullptr) {
             return;
         }
-        const auto secondsRemaining = duration - elapsed;
+        const auto secondsRemaining = duration - std::chrono::duration_cast<std::chrono::seconds>(elapsed);
```

Comment:
What will the result of such an operation be? Are we ok with rounding milliseconds down? For instance, 999ms will be converted to 0 seconds.

Reply:
I think its fine here. We are just deciding how many seconds to show to user when there is 1,999ms remaining. We can choose rounding up or down, I dont think it realy matters here. Especially since in this particular scenario we have 1s intervals, so it shouldnt be a problem.
Let me know if you think using rounding function will make this more clear to user, I have really no preference.